src/main/java/frc/robot/Constants.java

```diff
@@ -13,6 +13,11 @@
 
 public final class Constants {
     public static final double stickDeadband = 0.1;
+    public static final int shooterID = 0;
+    public static final int shooterServoID = 0;
+    public static final double shooterSpin = 0.7;
+    public static final int motorStop = 0;
+    public static final double turretPower = 0.1;
```

Comment:
Move these into the consolidated section at the bottom where @owenflatman made the sub classes.